web/modules/custom/joinup_core/tests/src/ExistingSite/NodeWorkflowTestBase.php

```diff
@@ -85,14 +89,14 @@ abstract class NodeWorkflowTestBase extends JoinupWorkflowTestBase {
   /**
    * {@inheritdoc}
    */
-  public function setUp() {
+  public function setUp(): void {
     parent::setUp();
 
-    $this->workflowAccess = $this->container->get('joinup_core.node_workflow_access');
+    $this->workflowAccess = \Drupal::service('joinup_core.node_workflow_access');
```

Comment:
This change is probably not required. I think the original version was probably better.